renovate.json

```diff
@@ -14,5 +14,27 @@
       "groupName": "docker images",
       "matchFileNames": [".docker/**"]
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^\\.deps/.*\\.yaml$"],
+      "matchStrings": [
+        "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\nversion: (?<currentValue>.*)"
```

Comment:
Since all of the deps there are github releases, we could use `datasourceTemplate` here, and set it to github releases. This way we don't need to set it in the custom annotation each time :)